backend/de.metas.business/src/main/java/de/metas/allocation/api/impl/AllocationDAO.java

```diff
@@ -90,6 +93,15 @@ public final BigDecimal retrieveOpenAmt(
 		return openAmt;
 	}
 
+	@Override
+	public final Money retrieveOpenAmt(
```

Comment:
consider renaming this method to `retrieveOpenAmtInInvoiceCurrency` like the other (deprecated) one.

about the deprecated one:
* consider renaming as `retrieveOpenAmtBDInInvoiceCurrency`
* consider removing it and replacing the calls with retrieveOpenAmtInInvoiceCurrency(...).toBigDecimal()